<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompanyEditDialog</class>
 <widget class="QDialog" name="CompanyEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменение поставщика</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Название организации</string>
   </property>
   <widget class="QPushButton" name="ChangeNameButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить название</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InitialNameInput">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewNameInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее название</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новое название организации</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Пользователь</string>
   </property>
   <widget class="QPushButton" name="ChangeUsernameButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить пользователя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InitialUsernameInput">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий поставщик</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новый поставщик</string>
    </property>
   </widget>
   <widget class="QComboBox" name="UsernameInputSelect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Склад</string>
   </property>
   <widget class="QPushButton" name="ChangeStoreButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить пользователя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InitialStoreInput">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий адрес</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новый склад</string>
    </property>
   </widget>
   <widget class="QComboBox" name="StoreSelect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
